pkg/pipelines/webhook/webhook.go

```diff
@@ -106,10 +110,41 @@ func newWebhookInfo(accessToken, pipelinesFile string, serviceName *QualifiedSer
 	if err != nil {
 		return nil, fmt.Errorf("failed to get event listener URL: %v", err)
 	}
-
+	if accessToken == "" {
```

Comment:
accessToken is first used in line 134 above.   Should we "get" access token before it?
Also, if the user provides accessToken, should we set it in keyring?